console/src/view/tasks.rs

```diff
@@ -260,6 +295,14 @@ impl List {
 
         frame.render_widget(Paragraph::new(controls), controls_area);
 
+        if let Some(area) = warnings_area {
+            let block = styles.border_block().title(Spans::from(vec![
+                bold("Warnings"),
```

Comment:
I quite like how this highlights warnings at a quick glance. This let's a user know what the warning is more quickly. I do wonder if a lot of tasks trigger a warning, if this extra table will be too big...

I had originally thought of adding another column to the regular table, that would show a ⚠️ in a row if the task had any. Then you could hit enter and see the task detail view to read about what warnings there were. Maybe that will help when there are a lot of warnings.

Reply:
hmm, that's a good point that there potentially might be a large number of warnings. we should probably also add the :warning: column to the tasks table (this would also let you sort by that column so that tasks with warnings are at the top of the list!). we could _also_ show warnings at the top like this, though...we could maybe have a maximum number of lines to use for it (say, 5), and if there's more than that, we could just show a message that says ":warning: 385 tasks have warnings! See the task details page for those tasks for more information." or something.

Reply:
eventually, when there are different classes of warnings, it would be cool to show a summary of each warning kind that was found and a count, if there are a lot of warnings. like
```
⚠ 201 tasks have woken themselves more than 50% of the time!
⚠ 3 tasks have never yielded for over 300 ms! Are those tasks blocking?
⚠ 519 tasks have been idle for more than 90% of their lifetime. Is the runtime over-saturated?
```

Reply:
(i made these up off the top of my head)